user-guide.adoc

```diff
@@ -6,6 +6,8 @@
 :product-name: Zimbra 9
 :document-title: User guide
 :copyright-year: 2020

```

Comment:
This top documentation file is "user-guide.adoc", but the README indicates that `index.adoc` should be used to produce the asciidoctor output.

The README should be updated, or this file renamed.